README.md

```diff
@@ -65,6 +65,7 @@ The following settings are supported:
 - `yaml.style.flowMapping` : Forbids flow style mappings if set to `forbid`
 - `yaml.style.flowSequence` : Forbids flow style sequences if set to `forbid`
 - `yaml.keyOrdering` : Enforces alphabetical ordering of keys in mappings when set to `true`. Default is `false`
+- `yaml.recommendations.show` : Recommend to install [OpenShift Toolkit](https://github.com/redhat-developer/vscode-openshift-tools) extension when detects devfile.yaml. Default is `true`
```

Comment:
We should probably seperate this into its own PR. 

The definition should be more general. We can have recommendation for different file in the future. Something like. "Enable extension recommendations for YAML files". 

Also do we have a chance to rename this to `yaml.extension.recommendations` ?